src/components/contact-govuk-one-login/index.njk

```diff
@@ -101,6 +101,7 @@
         {% endfor %}
       </ul>
       {% endif %}
+      <p class="govuk-body">{{ 'pages.contact.section1.guidance-for-helpers' | translate | safe | replace("[changePasswordHref]", changePasswordLink) }}</p>
```

Comment:
the translations don't have the placeholder `[changePasswordHref]` in them

Reply:
Removed.